modules/navigation/src/main/java/com/sygic/maps/module/navigation/NavigationFragment.kt

```diff
@@ -240,6 +254,24 @@ class NavigationFragment : MapFragmentWrapper<NavigationFragmentViewModel>(),
             }
         }.root
 
+    private fun showActionMenu(actionMenuData: ActionMenuData) {
```

Comment:
ako sme sa bavili, bolo by fajn vyviest na `NavigationFragment` metodu na nastavenie vlastnych `ActionMenuData` itemov

Reply:
itema by mohla mat lambdu a tym padom nemusi byt ani custom listener nastavovany userom

Reply:
je to tam, ale nakonec jsem přeci jen zvolil ten listener, protože lambdy bychom nemohli parcelovat :(